src/screens/charging-stations/connector-details/ChargingStationConnectorDetails.tsx

```diff
@@ -702,7 +718,7 @@ export default class ChargingStationConnectorDetails extends BaseAutoRefreshScre
       <Container style={style.container}>
         <HeaderComponent
           navigation={this.props.navigation}
-          title={chargingStation ? chargingStation.id : '-'}
+          title={chargingStation ? chargingStation.id.toString() : '-'}
```

Comment:
the 'id' is already a string, use 'as string' instead.

Reply:
The cast or the type conversion is then unneeded.

Reply:
We need somethign because id is of type 'string | number '

Reply:
I think then that the type definition is incorrect. The id type for `ChargingStation` is a string on the backend.

Reply:
yes I know but here we have a base class defining a common id attribute

Reply:
You can revert to the type cast then. Sorry for the noise.